Childrens-Social-Care-CPD/Views/PartialViews/Accordion.cshtml

```diff
@@ -0,0 +1,45 @@
+﻿@using Childrens_Social_Care_CPD.Constants
+@using Childrens_Social_Care_CPD.Enums
+@model List<RichText>;
+@addTagHelper *, Contentful.AspNetCore
+
+@if (Model.Any())
+{
+    <div class="govuk-accordion" data-module="govuk-accordion" id="accordion-default">
+        @{
+            var idCounter = 0;
+            var heraderId = "";
+            var contentId = "";
+            var summarySectionId = "";
+        }
+
+        @foreach (var accordionItem in Model)
+        {
+            idCounter++;
+            heraderId = $"accordion-default-heading-{@idCounter}";
+            contentId = $"accordion-default-content-{@idCounter}";
+            summarySectionId = $"accordion-with-summary-sections-summary-{idCounter}";
+            <div class="govuk-accordion__section">
+                <div class="govuk-accordion__section-header">
+                    <h2 class="govuk-accordion__section-heading">
+                        <span class="govuk-accordion__section-button" id=@heraderId>
+                            @accordionItem.Heading
+                        </span>
+                    </h2>
+                    <span class="govuk-accordion__section-summary govuk-body" id=@summarySectionId>
+                        <span class="govuk-accordion__section-summary-focus">
+                            @accordionItem.SubHeading
+                        </span>
+                    </span>
+                </div>
+                <div id=@contentId class="govuk-accordion__section-content" aria-labelledby=@heraderId>
+                    @{
+                        accordionItem.Heading = string.Empty;
```

Comment:
what is this doing, not sure why we are setting this as empty? If we dont need them why dont we have another model?

Reply:
@rgparkins  This is not a new model. Its a rich text, for this control variable is named as Rich text